source/stylesheets/screen.css.scss

```diff
@@ -91,3 +91,11 @@ ul.subdir__menu {
 .indented-list {
   margin: 0 0 0 20px;
 }
+
+.architecture-diagram {
+  height: 700px;
+  width: calc(100vw - 400px);
```

Comment:
And here. `calc(100vw - ((#{$govuk-gutter} * 2) + 330px))` (I'm not sure where the 330px width of the sidebar is defined)

Also not tested!